bzkanban.js

```diff
@@ -86,6 +88,7 @@ function initNav() {
 
     nav.appendChild(createQueryFields());
     if (bzAllowEditBugs) {
+        nav.appendChild(createBacklogButton());
         nav.appendChild(createBacklogTarget());
```

Comment:
Maybe we can get rid of the old backlog target now? I think intuitively now backlogging a card would be done by dragging it into the backlog column.